src/Orleans/Configuration/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Orleans.Runtime.Configuration
+{
+    public static class ConfigurationExtensions
```

Comment:
I think it's a bit easier to maintain when it's in the same class. There are legitimate cases for extension methods though - when there are extra dependencies, such as `AzureUtils`.